.evergreen/config.yml

```diff
@@ -2495,34 +2458,39 @@ buildvariants:
 - name: "Coverage Report"
   display_name: "Coverage Report"
   run_on:
-    - ubuntu1604-test
+    - ubuntu1804-test
   tasks:
      - name: "coverage-report"
   expansions:
     set_xtrace_on: on
 
 - matrix_name: "atlas-connect"
   matrix_spec:
-    platform: awslinux
-    python-version: *amazon1-pythons
+    platform: ubuntu-18.04
+    python-version: "*"
   display_name: "Atlas connect ${python-version} ${platform}"
   tasks:
     - name: "atlas-connect"
 
 - matrix_name: "serverless"
   matrix_spec:
-    platform: awslinux
-    python-version: *amazon1-pythons
+    platform: ubuntu-18.04
+    python-version: "*"
     auth-ssl: auth-ssl
     serverless: "*"
+  exclude_spec:
+   - platform: ubuntu-18.04
+     python-version: ["system-python3"]
```

Comment:
system-python3 wasn't in the amazon1-pythons list.

Reply:
Ack.